src/service/filterService.js

```diff
@@ -6,9 +6,9 @@
  */
 var configUtil = require('sb-config-util')
 
-function getChannelSearchString (callback) {
-  var searchString = configUtil.getConfig('CHANNEL_FILTER_QUERY_STRING')
-  callback(null, searchString)
+function getMetaSearchData (callback) {
```

Comment:
suggest renaming to `getMetadataFilterQuery`

Reply:
Done